semcore/table/src/style/table.shadow.css

```diff
@@ -106,50 +106,33 @@ SCellHead[sorting] {
     background-color: var(--intergalactic-table-th-primary-cell-hover, #e0e1e9);
   }
 
-  &:hover SSortWrapper {
-    display: block;
+  &:not([active]):hover SSortWrapper {
+    display: inline-flex;
+    margin-left: -16px;
   }
 }
 
 SCellHead[use='primary'][active] {
   background-color: var(--intergalactic-table-th-primary-cell-active, #e0e1e9);
 }
 
-SCellHead[active] SCellHeadIconAsc {
-  position: relative;
-}
-
-SCellHead[active] SCellHeadIconDesc {
-  position: relative;
-}
-
-SColumn[sortIconFloat] SSortWrapper {
-  width: 0;
-  flex-basis: 0;
-}
-
 SCellHead[active] SSortWrapper {
-  display: block;
-  width: 16px;
-  flex-basis: 20px;
+  display: inline-flex;
   margin-left: var(--intergalactic-spacing-1x, 4px);
 }
 
 SSortWrapper {
   display: none;
-  flex-shrink: 99999;
   position: relative;
-  flex-basis: 20px;
-  height: 0;
 }
 
-SSortWrapper:before {
+SSortWrapper::before {
   content: '';
   position: absolute;
-  top: -2px;
+  top: 0;
   right: 0;
-  width: 20px;
-  height: 16px;
+  width: calc(var(--intergalactic-spacing-1x, 4px) + 16px);
```

Comment:
and here shouldn't be design token value :sweat_smile: 
design tokens are not used for `width` and `height`